### ISSUE TITLE
Optics Pass I Part III Issues: Lens Issues

### ISSUE THREAD

**Issue by jenmaxwell** [isetbio/isetbio#339]
The following comments/issues are all related to functions contained in the Lens sub-directory:

get
	Should we move the notes section into the description? I think it would be more cohesive to do so...
Lens
	Optional key/values section needs to be filled out.
	Should we keep the commented out methods?

**Comment by tlian7** [comment]
(Not specifically for the lens class but this was the closest issue...)

I spent some time confused about manipulating the transmittance value in the oi.optics.lens class. It seems that there's some outdated code in opticsSet that acts specifically on an outdated transmittance field in an optics structure. Can we get rid of those lines? I think it would save future users some head scratching!

**Comment by wandell** [comment]
Good catch!

There is an odd situation in which we use the 'transmittance' slot for the diffraction limited optics and we use the 'lens' object, which is a human lens, for human optics.  So, we need to keep transmittance and lens around, and we need to do the set/get carefully. 

I edited the opticsSet/opticsGet functions with the goal of being more clear about this.  I don't think we can delete the 'transmittance scale' and 'transmittance wave' options, though.  We just need to check more carefully when we set/get that we are using the right model (diffraction or human).
